api-specificatie/zrc/openapi.yaml

```diff
@@ -1,83 +1,504 @@
+  license:
+    name: EUPL 1.2
+  version: '1'
+security:
+  - basic: []
 paths:
-  /zaken:
+  '/betrokkenen/{id}':
+    get:
+      operationId: organisatorischeeenheid_read
+      description: ''
+      responses:
+        '200':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OrganisatorischeEenheid'
+      tags:
+        - betrokkenen
+    parameters:
+      - name: id
+        in: path
+        description: A unique integer value identifying this Organisatorische eenheid.
+        required: true
+        schema:
+          type: integer
+  /klantcontacten:
     post:
-      summary: Aanmaken nieuwe zaak
-      operationId: zaak_create
+      operationId: klantcontact_create
       description: >-
-        Maak een ZAAK aan.
+        Registreer een klantcontact bij een ZAAK.
 
-        Indien geen zaakidentificatie gegeven is, dan wordt deze automatisch
+
+        Indien geen identificatie gegeven is, dan wordt deze automatisch
         gegenereerd.
+      responses:
+        '201':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/KlantContact'
+      tags:
+        - klantcontacten
       requestBody:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/Zaak'
+              $ref: '#/components/schemas/KlantContact'
+        required: true
+    parameters: []
+  '/klantcontacten/{id}':
+    get:
+      operationId: klantcontact_read
+      description: Geef de details van een klantcontact voor een ZAAK.
       responses:
-        201:
-          description: "Representatie van aangemaakte ZAAK"
+        '200':
+          description: ''
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Zaak'
-        422:
-          description: Opgegeven zaak informatie kan niet verwerkt worden
+                $ref: '#/components/schemas/KlantContact'
+      tags:
+        - klantcontacten
+    parameters:
+      - name: id
+        in: path
+        description: A unique integer value identifying this klantcontact.
+        required: true
+        schema:
+          type: integer
+  /rollen:
+    post:
+      operationId: rol_create
+      description: Koppel een BETROKKENE aan een ZAAK.
+      responses:
+        '201':
+          description: ''
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Fout'
-        default:
-          description: Onverwachte fout tijdens opvoeren zaak
+                $ref: '#/components/schemas/Rol'
+      tags:
+        - rollen
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Rol'
+        required: true
+    parameters: []
+  '/rollen/{id}':
```

Comment:
Iin het RGBZ2 heeft een rol geen identificatie. Het is een relatieklasse die bestaat bij de gratie van een relatie tussen een specifieke betrokkene en een zaak.
De ID is deze API is dus puur een technisch ID, geen gegeven uit het RGBZ2. Zou dit iets in een UGM moeten worden? @zomwoof